File: UNIDAD 3/JAVA/U3_P21.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3_p21;
import java.util.Scanner;
/**
 *
 * @author fatim
 */
public class U3_P21 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num, NT,I=1;
        System.out.print("Ingresa el total de numeros: ");
        NT = sc.nextInt();
        while (I<=NT){
            System.out.print("Ingresa un numero: ");
            num = sc.nextInt();
                        
            if (num > 0)
                System.out.println("El numero " + num + " elevado al cubo es: " + (num*num*num));
            I++;

        }
    }
}

File: UNIDAD 3/C#/U3_P21.cs
using System;
namespace U3_P21
{
    internal class U3_P21
    {
        static void Main(string[] args)
        {
            int num, NT, I = 1;
            Console.WriteLine("Ingresa el total de numeros: ");
            NT = Convert.ToInt32(Console.ReadLine());
            while (I <= NT)
            {
                Console.WriteLine("Ingresa un numero: ");
                num = Convert.ToInt32(Console.ReadLine());

                if (num > 0)
                    Console.WriteLine("El numero " + num + " elevado al cubo es: " + (num * num * num));
                I++;

            }
           




        }
    }
}